client/src/assets/icons/green-not-completed.tsx

```diff
@@ -1,20 +1,27 @@
 import React from 'react';
 import { useTranslation } from 'react-i18next';
 
-function GreenNotCompleted(
-  props: JSX.IntrinsicAttributes & React.SVGProps<SVGSVGElement>
-): JSX.Element {
-  const { t } = useTranslation();
+interface GreenNotCompletedProps
+  extends JSX.IntrinsicAttributes,
+    React.SVGProps<SVGSVGElement> {
+  suppressSrOnly: boolean;
+}
 
+function GreenNotCompleted(props: GreenNotCompletedProps): JSX.Element {
+  const { t } = useTranslation();
+  const { suppressSrOnly, ...rest } = props;
```

Comment:
```suggestion
interface GreenNotCompletedProps
  extends JSX.IntrinsicAttributes,
    React.SVGProps<SVGSVGElement> {
  suppressSrOnly?: boolean;
}

function GreenNotCompleted(props: GreenNotCompletedProps): JSX.Element {
  const { t } = useTranslation();
  const { suppressSrOnly = false, ...rest } = props;
```

So the linter is angry about some instances of this component not having the property.

I think, if we only want to explicitly set it to `true`, we can use a default value of `false`. Otherwise, we'll need to pass a value in the flagged component calls.

Reply:
I quite like making it required so that it's a conscious choice to hide elements from screenreaders or not.

However, I think we should simplify the props.  In essence, both props hide elements from the screenreader, so how about just calling them both `screenReaderHidden`?

Reply:
I unified the prop name and hopefully found we one can all agree on? I think `hushScreenReaderText` does a good job of explaining what this prop does. Since these two icons are used all over the place, their default behavior needs to be left intact, and that behavior is to include the sr-only text, so this prop needs to be `false` by default.